src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -125,7 +140,6 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
-        c.argument('sku', help='The sku of the App Configuration store', arg_type=get_enum_type(['Free', 'Standard']))
```

Comment:
Why did we move sku from the common section to appconfig create and appconfig update sections?

Reply:
Oh is it because we dont want to validate the sku during update operation?

Reply:
Yeah, update operation doesn't need the replication validation